bootstrap.sh

```diff
@@ -0,0 +1,40 @@
+# Set the path for the file we are going to generate
+OUTPUT="$(pwd)/setup.txt"
+
+# Figure out what OS we are on
+current_os="$(uname -s)"
+case "${current_os}" in
+  Linux*)     machine=Linux;;
+  Darwin*)    machine=Mac;;
+  CYGWIN*)    machine=Cygwin;;
+  MINGW*)     machine=MinGw;;
+  *)          machine="UNKNOWN:${current_os}"
+esac
+
+# this is a utility to help adding more variables to the generated file
+append_line() {
+  echo $1 >> $OUTPUT
+}
+
+# Based on the operating system get the correct vendor path
+# for the Oni2 binary
+case "${machine}" in
+  Linux)
+      ONI_PATH="$(pwd)/vendor/neovim-0.3.3/nvim-linux64/bin/nvim";;
+  Darwin)
+      ONI_PATH="$(pwd)/vendor/neovim-0.3.3/nvim-osx64/bin/nvim";;
+  *)
+      ONI_PATH="$(pwd)/vendor/neovim-0.3.3/nvim-win64/bin/nvim.exe"
+      ONI_PATH="$(cygpath -m $ONI_PATH)";;
```

Comment:
There were a few bugs w/ this on Windows - I tweaked it a bit in https://github.com/onivim/oni2/pull/43/commits/0a3f6ba1dc4a641b1fdf58bdae1e51533eb625b6 👍

Reply:
Thanks for trying it out and applying a fix :100: